feature/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Try to keep an empty blank line after every file

Reply:
Only one entry on .gitignore right now but will keep that in mind.